scripts/js/lib/markdownTitles.ts

```diff
@@ -0,0 +1,66 @@
+import { unified } from "unified";
```

Comment:
You need to add the copyright comment to the top of this file (just copy from another script and update the year)